src/visualization/manipulator.cc

```diff
@@ -288,7 +290,6 @@ bool FreeManipulator::mouseEvent(QObject* watched, QMouseEvent* event) {
         }
         prev_pos_ = event->pos();
       }
-      return true;

```

Comment:
As discussed privately: it doesn't matter in this case, but we may change this in the future.